src/libutil/file-system.cc

```diff
@@ -611,55 +605,45 @@ static void setWriteTime(const fs::path & p, const struct stat & st)
 }
 #endif
 
-void copy(const fs::directory_entry & from, const fs::path & to, bool andDelete)
```

Comment:
I think we might still want a `copyEntry` function as an implementation detail because it can avoid another stat in many cases (because the directory listing already has the info).